script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -179,21 +179,21 @@ I mean, even with a gag order, there's going to be leaks, right?"
 -- Page 798, Offset 13013.
 -- 「……アレでもちっと緊張感があれば使えるんだがな。あの捨て鉢さはなんだ、若い頃に挫折でもしたのかカリウス？」
 "...That guy would really come in handy if he had %{i}any%{/i} fucking sense of urgency at all.
- How'd he end up in such a sorry state...
+ How'd he end up in such a sorry state...?
  His spirit get crushed when he was young or something, Karius?"
 }
 [sha:c99b4827f10eb2502f08740495726839c3e10818]{
 -- Page 799, Offset 13014.
 -- 「私にはとんと。安藤の私生活に興味はありませんので。
-"I haven't the slightest idea. Andou's private life holds no interest for me.
+"I haven't the slightest idea. Andou's private life holds no interest to me.
 }
 [sha:4d5e1406f183d7f3cc5b7de59cefb742bdfb0098]{
 -- Page 799, Offset 13015.
 -- ３年前、悪徳司祭の内部告発を決意したものの、最後の最後で金に目が眩んだ恋人に裏切られ、故郷を追われ、失意の果てに父方の祖国であるこの国に島流しになった、という事ぐらいしか。
 All I know is that three years ago,
  he decided to blow the whistle on a corrupt presbyter,
- but was betrayed at the final hour by his lover, who had been blinded by greed,
- after which he was driven from his hometown before,
+ but was betrayed at the final hour by his lover, who had been blinded by greed.
```

Comment:
ross had wanted to keep this one a horrible run on sentence on purpose.

Reply:
Yeah this sentence is meant to be run-on. It's part of the joke.